server/controllers/finance/cost_center.js

```diff
@@ -276,6 +276,49 @@ function del(req, res, next) {
     .done();
 }
 
+// This function searches for account membership at a cost center,
+// and returns an object table that will allow when creating a transaction
```

Comment:
I found this description confusing.   Could you re-write it and explain what you are trying to do in a little more detail?